src/include/sof/lib/dai.h

```diff
@@ -202,7 +219,11 @@ static inline int dai_pm_context_restore(struct dai *dai)
 static inline int dai_get_handshake(struct dai *dai, int direction,
 				    int stream_id)
 {
-	return dai->drv->ops.get_handshake(dai, direction, stream_id);
+	int ret = dai->drv->ops.get_handshake(dai, direction, stream_id);
+
+	platform_shared_commit(dai, sizeof(*dai));
```

Comment:
I feel like dai_get_handshake is a read-only operation so it technically doesn't need a commit?